jobDataCatalogue.py

```diff
@@ -0,0 +1,128 @@
+from job import Job
+import logging
+
+log = logging.getLogger(__name__)
+
+
+class JobDataCatalogue(Job):
+    """
+    Class to copy cohort rich metadata and cohort and network cdm metadata from data catalogue
+    cohort staging areas to catalogue.
+    """
+
+    def __init__(self, url, email, password, catalogueDB, sourceDB):
+        super().__init__(url, email, password, catalogueDB, sourceDB)
+
+        self.cohortPid = self.fetchCohortPid(self.staging, self.sourceDB)
+        self.modelPid = self.fetchModelPid(self.staging, self.sourceDB)
+
+        if self.sourceDB == 'SharedStaging':
+            self.syncShared()
+        elif self.cohortPid is None and self.modelPid is None:
+            log.info('Skip sync for: ' + self.sourceDB)
+        elif self.cohortPid:
+            self.syncCohort()
+        elif self.modelPid:
+            self.syncNetwork()
+
```

Comment:
suggest to remove job action from constructor function and into explicit 'run' function